src/EnergyPlus/DataAirSystems.hh

```diff
@@ -214,204 +189,146 @@ namespace DataAirSystems {
+        int SupMixInNode = 0;                                 // node num of supply air inlet to mixer
+        int MixOutNode = 0;                                   // outlet node of mixer
+        int RABSplitOutNode = 0;                              // node num of RAB splitter outlet
+        int OtherSplitOutNode = 0;                            // node num of nonRAB splitter outlet
+        int NumOACoolCoils = 0;                               // number of cooling coils in the outside air system
+        int NumOAHeatCoils = 0;                               // number of heating coils in the outside air system
+        int NumOAHXs = 0;                                     // number of heat exchangers in the outside air system
+        bool SizeAirloopCoil = true;                          // simulates air loop coils before calling controllers
+        FanModelType supFanModelType = FanModelType::Invalid; // indicates which type of fan model to call for supply fan, legacy or new OO
+        int SupFanNum = 0;       // index of the supply fan in the Fan data structure when model type is StructArrayLegacyFanModels
+        int supFanVecIndex = -1; // index in fan object vector for supply fan when model type is ObjectVectorOOFanSystemModel, zero-based index
+        FanPlacement supFanLocation = FanPlacement::Invalid;  // location of fan relative to coil
+        FanModelType retFanModelType = FanModelType::Invalid; // indicates which type of fan model to call for return fan, legacy or new OO
+        int RetFanNum = 0;           // index of the return fan in the Fan data structure when model type is StructArrayLegacyFanModels
+        int retFanVecIndex = -1;     // index in fan object vector for return fan when model type is ObjectVectorOOFanSystemModel, zero-based index
+        Real64 FanDesCoolLoad = 0.0; // design fan heat gain for the air loop [W]
     };
 
     struct ConnectionPoint
     {
         // Members
-        int LoopType;
-        int LoopNum;
-        int BranchNum;
-        int CompNum;
-
-        // Default Constructor
-        ConnectionPoint() : LoopType(0), LoopNum(0), BranchNum(0), CompNum(0)
-        {
-        }
+        int LoopType = 0;
+        int LoopNum = 0;
+        int BranchNum = 0;
+        int CompNum = 0;
     };
 
     struct ConnectZoneComp
     {
         // Members
-        int ZoneEqListNum;
-        int ZoneEqCompNum;
-        int PlantLoopType;
-        int PlantLoopNum;
-        int PlantLoopBranch;
-        int PlantLoopComp;
-        int FirstDemandSidePtr;
-        int LastDemandSidePtr;
-
-        // Default Constructor
-        ConnectZoneComp()
-            : ZoneEqListNum(0), ZoneEqCompNum(0), PlantLoopType(0), PlantLoopNum(0), PlantLoopBranch(0), PlantLoopComp(0), FirstDemandSidePtr(0),
-              LastDemandSidePtr(0)
-        {
-        }
+        int ZoneEqListNum = 0;
+        int ZoneEqCompNum = 0;
+        int PlantLoopType = 0;
```

Comment:
:pushpin: we just defined `struct ConnectionPoint` and now here is a logical instance of that `struct` and ... we're not using it.